src/Raven.Studio/typescript/viewmodels/database/indexes/editIndex.ts

```diff
@@ -544,6 +553,30 @@ class editIndex extends shardViewModelBase {
             });
     }
 
+    private fetchDatabaseSettings() {
+        eventsCollector.default.reportEvent("database-settings", "get");
+        
+        return new getDatabaseSettingsCommand(this.db)
```

Comment:
How about getting this data somewhere else instead of pushing it in through the constructor multiple times?